src/main/java/edu/wpi/grip/ui/pipeline/input/RangeInputSocketView.java

```diff
@@ -64,7 +67,17 @@ public RangeInputSocketView(EventBus eventBus, InputSocket<List<Number>> socket)
 
         this.slider.disableProperty().bind(this.getHandle().connectedProperty());
 
-        this.setContent(this.slider);
+        // Add a label under the slider to show the exact range
+        final Label label = new Label(String.format("%.0f - %.0f", initialLow, initialHigh));
+        label.setMaxWidth(Double.MAX_VALUE);
+        label.setAlignment(Pos.CENTER);
+        this.slider.lowValueProperty().addListener(observable ->
+                label.setText(String.format("%.0f - %.0f", this.slider.getLowValue(), this.slider.getHighValue())));
```

Comment:
Can't you use a binding converter for this? I know that numbers have a `.multiply()` or `.add()` binding. I think that strings have the same thing but I could be wrong.